### DISCUSSION TITLE
What happens when I reach my hobby plan limits?

### DISCUSSION THREAD

**Discussion by jaywonchung** [vercel/community#6068] [Feature Question] [answered]
### Question

I started hosting a service in Vercel under the hobby plan, and I was wondering what happens when a metric (ilke bandwidth) exceeds the hobby plan limit.
Does the account or deployment automatically get paused until the next month, or will the deployment stay alive and will I receive an invoice for additional usage?

Thanks a lot.

**Comment by amyegan** [comment, marked as answer]
Good question @jaywonchung. The Hobby plan does not provide pay-as-you-go usage, so deployments could be paused until metrics reset and usage is freed up. You can learn more on our [Fair Use Guidelines](https://vercel.com/docs/limits/fair-use-guidelines#) and [Limits](https://vercel.com/docs/limits/overview#) pages.

**Reply by jaywonchung** [reply]
Thanks!
